server/src/schema.js

```diff
@@ -3,6 +3,10 @@ const { pubSub } = require('./subscriptions')
 const { conflictHandler } = require("@aerogear/voyager-conflicts")
 
 const typeDefs = gql `
+
+directive @hasRole(role: [String]) on FIELD | FIELD_DEFINITION
```

Comment:
I thought we no longer need that.

Reply:
OK. My bad: https://github.com/aerogear/voyager-server/blob/master/examples/keycloak/server.js#L18